### PR TITLE
Implemented create svg schematic text function 

### PR DESCRIPTION
**Pull request by anas-sarkez**

### CODE REVIEW

File: lib/sch/convert-circuit-json-to-schematic-svg.ts [hunk 1]
```diff
@@ -120,8 +122,13 @@ export function convertCircuitJsonToSchematicSvg(
     } else if (elm.type === "schematic_trace") {
       schTraceSvgs.push(...createSchematicTrace(elm, transform))
     } else if (elm.type === "schematic_net_label") {
+      console.log(elm, transform)
```
Comment (seveibar) [inline lib/sch/convert-circuit-json-to-schematic-svg.ts:125]:
```suggestion
```

File: lib/sch/convert-circuit-json-to-schematic-svg.ts [hunk 2]
```diff
@@ -120,8 +122,13 @@ export function convertCircuitJsonToSchematicSvg(
...
       schNetLabel.push(...createSvgObjectsForSchNetLabel(elm, transform))
     }
+    else if (elm.type === "schematic_text") {
+      console.log(elm, transform)
```
Comment (seveibar) [inline lib/sch/convert-circuit-json-to-schematic-svg.ts:129]:
```suggestion
```

File: lib/sch/svg-object-fns/create-svg-objects-for-sch-text.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  transform: Matrix
+): SvgObject => {
+
+
+  // Apply transformation
+  const center = applyToPoint(transform, elm.position);
+
+  console.log("Transformed Center Position:", center);
+  console.log("Element Position:", elm.position);
```
Comment (seveibar) [inline lib/sch/svg-object-fns/create-svg-objects-for-sch-text.ts:24]: Check for logs

**Review by seveibar — CHANGES_REQUESTED** [review]
This PR is _great_, but for prs introducing this type of function the snapshot test _must_ be rendered with grid with some labeled points (check out the net label snapshot) because if we dont check the points we often find out that the svg is off by a bit.

after that looks good!!!

**Comment by anas-sarkez** [comment]
@seveibar checkout the snapshot

**Review by seveibar — APPROVED** [review]
Make sure to run format

**Pull request merged**